src/goRename.ts

```diff
@@ -22,17 +22,17 @@ export class GoRenameProvider implements vscode.RenameProvider {
 			var filename = this.canonicalizeForWindows(document.fileName);
 			var offset = document.offsetAt(position);
 
-			var gorename = path.join(process.env["GOPATH"], "bin", "gorename");
+			var gorename = getBinPath("gorename");
 
 			cp.execFile(gorename, ["-offset", filename + ":#" + offset, "-to", newName], {}, (err, stdout, stderr) => {
 				try {
 					if (err && (<any>err).code == "ENOENT") {
 						vscode.window.showInformationMessage("The 'gorename' command is not available.  Use 'go get golang.org/x/tools/cmd/gorename' to install.");
-						return resolve(null);
+						return Promise.resolve<vscode.WorkspaceEdit>(null);
```

Comment:
Actually both of these should be `resolve(null)`.  The `return` itself doesn't matter really (just short circuits the exit from the callback).  The call to the `resolve` from the `new Promise()` above is what ensures a value is being returned from the promise.  (aside - moving to async/await will dramatically improve a lot of this code - hoping to do that soon now that TS supports this with the same Node version as VS Code uses)